notes/bugfix-296.md

```diff
@@ -0,0 +1,4 @@
+# Autofoucs on message box when typing
+
+If typing and there is no focussed control and the keyDown message is
+passed to the IDE backscript the Message Box will focus
```

Comment:
I think you mean "Autofocus" rather than "Autofoucs".

Is there a way you could describe the change in a less technical but still accurate way? Probably most of our users don't care about the fact the IDE has a backscript but do care that they can mostly just start typing at random bits of the IDE and the keystrokes will end up in the message box.